packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Breaking Change
+* Update to terra-list v4, the result of themeable variables changing
```

Comment:
Do we need to include the to the upgrade guide?

Reply:
Updated.